src/attributes/components/AttributeValueEditDialog/AttributeValueEditDialog.tsx

```diff
@@ -45,47 +57,66 @@ const AttributeValueEditDialog: React.FC<AttributeValueEditDialogProps> = ({
 }) => {
   const intl = useIntl();
   const initialForm: AttributeValueEditDialogFormData = {
-    name: maybe(() => attributeValue.name, "")
+    name: attributeValue?.name ?? "",
+    slug: attributeValue?.slug ?? "",
+    value: attributeValue?.value ?? ""
   };
+
+  const fields = [
+    {
+      message: M.dialogMessages.name,
+      field: "name"
+    },
+    {
+      message: M.dialogMessages.adminName,
+      field: "slug"
+    },
+    {
+      message: M.dialogMessages.attributeValue,
+      field: "value"
+    }
+  ];
   const errors = useModalDialogErrors(apiErrors, open);
-  const formErrors = getFormErrors(["name"], errors);
+  const formErrors = getFormErrors(["name", "value"], errors);
 
   return (
     <Dialog onClose={onClose} open={open} fullWidth maxWidth="sm">
       <DialogTitle>
-        {attributeValue === null ? (
-          <FormattedMessage
-            defaultMessage="Add Value"
-            description="add attribute value"
-          />
-        ) : (
-          <FormattedMessage
-            defaultMessage="Edit Value"
-            description="edit attribute value"
-          />
-        )}
+        {attributeValue === null ? M.addValue : M.editValue}
       </DialogTitle>
-      <Form initial={initialForm} onSubmit={onSubmit}>
-        {({ change, data, submit }) => (
+      <Form
+        initial={initialForm}
+        onSubmit={data => onSubmit(omit(data, ["slug"]))}
+      >
+        {({ change, data, submit, set }) => (
           <>
             <DialogContent>
-              <TextField
-                autoFocus
-                disabled={disabled}
-                error={!!formErrors.name}
-                fullWidth
-                helperText={getAttributeValueErrorMessage(
-                  formErrors.name,
-                  intl
-                )}
-                name={"name" as keyof AttributeValueEditDialogFormData}
-                label={intl.formatMessage({
-                  defaultMessage: "Name",
-                  description: "attribute name"
-                })}
-                value={data.name}
-                onChange={change}
-              />
+              {fields.map(({ message, field }, idx) => (
+                <React.Fragment key={idx}>
+                  {idx > 0 && <FormSpacer />}
+                  <TextField
+                    autoFocus={idx === 0}
+                    disabled={disabled || field === "slug"}
+                    error={!!formErrors[field]}
+                    fullWidth
+                    helperText={getAttributeValueErrorMessage(
+                      formErrors[field],
+                      intl
+                    )}
+                    name={field as keyof AttributeValueEditDialogFormData}
+                    label={intl.formatMessage(message)}
+                    value={data[field]}
+                    onChange={evt => {
```

Comment:
We call it `event` everywhere else, let's stick to that. Also, please move onChange outside of return